input/pagecontent/index.md

```diff
@@ -1,5 +1,7 @@
 <div markdown="1" class="note-to-balloters">
 Voters familiar with mCODE can focus their attention on the changes since STU 3, listed in the [Release Notes](change_log.html).
+
+The authors are asking for feedback on the design approach for representing specific systems (e.g., specific cancer staging systems and performance scales). For example, currently, there are two "basic" profiles for Cancer Stage and Cancer Risk Assessment, with 8 additional profiles for specific staging systems and risk assessments that use the "basic" profiles as the parent. The pro of this approach is that these specific profiles provide detailed guidance on how to capture and represent data relevant to those specialized staging systems and risk assessments, helping to ensure better communication of data. However, the con of this approach is that it opens the question of which staging systems and risk assessments should be included as their own profiles and runs the risk of many, many profiles being created for each cancer stage system and risk assessment system that exist. An alternate option could be to keep the "basic" Cancer Stage and Cancer Risk Assessment profiles, remove the specific stage system and risk assessment profiles, but add narrative and examples that explain how common staging systems and risk assessments should be represented using the "basic" Cancer Stage and Cancer Risk Assessment profiles. We welcome your feedback.
```

Comment:
I recommend substituting 'advantage' for 'pro', and 'disadvantage' for 'con'.

Reply:
My suggestion:
The authors are asking for feedback on the design approach for representing cancer staging systems and risk assessments. Currently, there are two “basic” (abstract?) profiles for Cancer Stage and Cancer Risk Assessment, with 8 additional profiles for specific staging systems and risk assessments that use the “basic” profiles as the parent. The advantage of these tailored profiles help ensure more consistent communication of this data by providing detailed guidance on the FHIR representation of the data from these assessments.  However, this specificity comes at the expense of the lack of scalability of the approach, since a vast proliferation of profiles would be required to represent all cancer stage systems and risk assessments currently in use.   An alternate option could be to keep the “basic” Cancer Stage and Cancer Risk Assessment profiles, remove the specific stage system and risk assessment profiles, but add narrative and examples that explain how common staging systems and risk assessments should be represented using the “basic” Cancer Stage and Cancer Risk Assessment profiles. We welcome your feedback.